x-pack/plugins/alerting/common/rule.ts

```diff
@@ -213,7 +213,10 @@ export type SanitizedRule<Params extends RuleTypeParams = never> = Omit<
 > & { actions: SanitizedRuleAction[] };
 
 export type ResolvedSanitizedRule<Params extends RuleTypeParams = never> = SanitizedRule<Params> &
-  Omit<SavedObjectsResolveResponse, 'saved_object'>;
+  Omit<SavedObjectsResolveResponse, 'saved_object'> & {
```

Comment:
`ResolvedSanitizedRule` is referred to in so many places across the stack that removing it entirely is out of scope of one PR. Adding these props to it is necessary to prevent cascading typecheck failures.